policy_tests/tests/Makefile.bare64

```diff
@@ -0,0 +1,2 @@
+include isp-runtime-bare64.mk
```

Comment:
Didn't @Hackmodford 's changes remove the need for multiple Makefiles?

Reply:
Those changes were the ones holding things up.  By the time we get these reviews done, they might be ready though.

Reply:
They are now ready.